### PR TITLE
Feature/log package split

### PR DESCRIPTION
**Pull request by julianzimmermann**

### CODE REVIEW

File: codeception.yml [hunk 1]
```diff
@@ -59,6 +59,7 @@ include:
   - bundles/jellyfish-thirty-five-up
   - bundles/limit-buyable-quantity
   - bundles/log-io
+  - bundles/log-slack-connector
```
Comment (daniel-rose) [inline codeception.yml:62]:
```suggestion
  - bundles/log-slack-connector
  - bundles/log-aws-cloud-watch-connector
  - bundles/log-gelf-connector
```

File: composer.json [hunk 2]
```diff
@@ -23,7 +23,9 @@
     "fond-of-spryker/country": "dev-spryker_upgrade || dev-feature/paypal-refund",
     "fond-of-spryker/product-list-company": "dev-spryker_upgrade",
     "fond-of-spryker/product-list-customer": "dev-spryker_upgrade",
+    "graylog2/gelf-php": "^1.5.5",
     "hackzilla/password-generator": "^1.6",
+    "maxbanton/cwh": "^2.0",
```
Comment (daniel-rose) [inline composer.json:28]:
```suggestion
    "maxbanton/cwh": "^2.0.0",
```

File: bundles/log-aws-cloud-watch-connector/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Log AWS CloudWatch Connector
```
Comment (daniel-rose) [inline bundles/log-aws-cloud-watch-connector/README.md:1]: Delete Readme

File: bundles/log-aws-cloud-watch-connector/composer.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Julian Hyatt",
+      "email": "info@julianzimmermann.dev"
+    }
+  ],
+  "require": {
+    "php": ">=7.3",
+    "maxbanton/cwh": "^2.0",
```
Comment (daniel-rose) [inline bundles/log-aws-cloud-watch-connector/composer.json:13]:
```suggestion
    "maxbanton/cwh": "^2.0.0",
```

File: bundles/log-gelf-connector/README.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Log Gelf Connector
```
Comment (daniel-rose) [inline bundles/log-gelf-connector/README.md:1]: Delete readme

File: bundles/log-slack-connector/README.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Log Slack Connector
```
Comment (daniel-rose) [inline bundles/log-slack-connector/README.md:1]: Delete readme

**Review by daniel-rose — APPROVED** [review]

**Review by sourcecube — APPROVED** [review]

**Pull request merged**